problem_builder/dashboard.py

```diff
@@ -466,6 +470,26 @@ def add_error(msg):
                         .format(key=key, value=value)
                     )
 
+                if key not in data.mentoring_ids:
+                    add_error(
+                        _(u"Exclude questions is malformed: mentoring url_name {url_name} is not added to Dashboard")
+                        .format(url_name=key)
+                    )
+
+        if data.average_labels:
+            for key, value in data.average_labels.iteritems():
+                if not isinstance(value, basestring):
+                    add_error(
+                        _(u"Average Labels is malformed: value for key {key} is {value}, expected string")
```

Comment:
This reads weirdly in English. Recommend doing, `"'average_labels' is malformed"`